source/quick-start.html.md.erb

```diff
@@ -21,7 +21,7 @@ You have 3 different options to create an example service, depending on your nee
 | With the GOV.UK One Login simulator using source code.    | 10 minutes       | You'll see the simulated response from GOV.UK One Login and view additional code.        |
 | Using the GOV.UK One Login integration environment.       | 20 minutes       | You can use test user data to interact with the integration environment.                 |
 
-There's further guidance on using the [GOV.UK One Login simulator](https://docs.sign-in.service.gov.uk/gov-uk-one-login-simulator/) to test your service before you use the GOV.UK One Login integration environment.
+There's further guidance on using the [GOV.UK One Login simulator](https://docs.sign-in.service.gov.uk/test-your-integration/gov-uk-one-login-simulator/) to test your service before you use the GOV.UK One Login integration environment.
```

Comment:
@SteveEvans-GDS replace the absolute url reference with a relative reference to `/test-your-integration/gov-uk-one-login-simulator/`